### ISSUE TITLE
Compatibility with Ubuntu 16.10 / GNOME Shell 3.20.4

### ISSUE THREAD

**Issue by damienalexandre** [petres/gnome-shell-extension-services-systemd#16]
Hi there,
I just updated my system to Ubuntu 16.10 / GNOME Shell 3.20.4 and I see that this extension does not work anymore:

![image](https://cloud.githubusercontent.com/assets/225704/21761854/0319777a-d656-11e6-82a6-8110d9fdbcdf.png)

How can I help you to make it work on 16.10?

**Comment by zapashcanon** [comment]
```sh
gsettings set org.gnome.shell disable-extension-version-validation "true"
```

**Comment by damienalexandre** [comment]
Yes, what is strange is that the `metadata.json` file already contains 3.20:

```
	"shell-version": [
		"3.20", "3.22"
	]
```

**Comment by zapashcanon** [comment]
I guess the version uploaded to GNOME Shell website is older...
